UnitsNet.Tests/QuantityTests.cs

```diff
@@ -51,5 +51,28 @@ private static void AssertQuantityInfoRepresentsLength(QuantityInfo<LengthUnit>
             Assert.Equal(lengthUnitNames, quantityInfo.UnitNames);
 #pragma warning restore 618
         }
+
+        [Fact]
+        public void TestGenericMultiplication()
+        {
+            var length1 = Length.FromInches(2.0);
+            var length2 = Length.FromMeters(2.0);
```

Comment:
Can we add an example of multiplying different quantities, such as `Force * Length = Torque`.

Reply:
I removed this entirely. Was a test method for me more than anything. I'll do this in another PR once this is in